packages/tools/fluid-runner/src/exportFile.ts

```diff
@@ -56,22 +56,25 @@ export async function exportFile(
 			logger,
 			{ eventName: "ExportFile" },
 			async () => {
-				const argsValidationError = getArgsValidationError(inputFile, outputFile);
+				const argsValidationError = getArgsValidationError(inputFile, outputFile, timeout);
 				if (argsValidationError) {
 					const eventName = clientArgsValidationError;
 					logger.sendErrorEvent({ eventName, message: argsValidationError });
 					return { success: false, eventName, errorMessage: argsValidationError };
 				}
 
-				fs.writeFileSync(
-					outputFile,
-					await createContainerAndExecute(
+				const executeResult = await timeoutPromise<string>((resolve, reject) => {
+					createContainerAndExecute(
 						getSnapshotFileContent(inputFile),
 						fluidFileConverter,
 						logger,
 						options,
-					),
-				);
+					)
+						.then((value) => resolve(value))
+						.catch((error) => reject(error));
```

Comment:
error will have some info about timeout, right?

Reply:
This error would be what `createContainerAndExecute` potentially throws